hack/test-go.sh

```diff
@@ -21,6 +21,8 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+trap 'exit 1' SIGINT
```

Comment:
I really don't understand why... exit 1 is pretty much the default for Ctrl+C, what am I missing here?